packages/templates/src/getTemplates.js

```diff
@@ -20,6 +20,70 @@
 import { getTimeTracker } from '@web-stories-wp/tracking';
 import { DATA_VERSION, migrate } from '@web-stories-wp/migration';
 
+const TEMPLATE_NAMES = [
+  'fresh-and-bright',
+  'food-and-stuff',
+  'doers-get-more-done',
+  'weekly-entertainment',
+  'fashion-on-the-go',
+  'no-days-off',
+  'experience-thailand',
+  'sleep',
+  'baking-bread-guide',
+  'sangria-artichoke',
+  'ways-to-eat-avocado',
+  'kitchen-stories',
+  'album-releases',
+  'almodos-films',
+  'pizzas-in-nyc',
+  '12-hours-in-barcelona',
+  'ultimate-comparison',
+  'fitness-apps-ranked',
+  'street-style-on-the-go',
+  'plant-based-dyes',
+  'indoor-garden-oasis',
+  'belly-fat-workout',
+  'tv-show-recap',
+  'pride-month-watchlist',
+  'honeymooning-in-italy',
+  'ace-hotel-kyoto-review',
+  'how-video-calls-saved-the-day',
+  'laptop-buying-guide',
+  'beauty-quiz',
+  'diy-home-office',
+  'kitchen-makeover',
+  'self-care-guide',
+  'rock-music-festival',
+  'celebrity-q-and-a',
+  'los-angeles-city-guide',
+  'hawaii-travel-packing-list',
+  'google-music-studio-tour',
+  'how-contact-tracing-works',
+  'summer-fashion-collection',
+  'buying-art-on-the-internet',
+  'house-hunting',
+  'new-york-party-round-up',
+  'a-day-in-the-life',
+  'elegant-travel-itinerary',
+  'modernist-travel-guide',
+  'simple-tech-tutorial',
+  'magazine-article',
+  'fashion-inspiration',
+  'skin-care-at-home',
+  'art-books-gift-guide',
+  'vintage-chairs-buying-guide',
+];
+
+export function getTemplateCreationDates() {
+  return Promise.all(
+    TEMPLATE_NAMES.map((title) =>
+      import(
+        /* webpackChunkName: "chunk-web-stories-template-[index]-creationDate" */ `./raw/${title}/creationDate`
+      ).then((data) => [title, data.default])
+    )
```

Comment:
I see why you created separate `creationDate.js` files, but it seems a bit odd to have all these extra files just exporting a string.

What if I want a list of all template names (not slugs) in the future? Or a list of all templates & their colors? I'm not gonna create a new file for each value.

Wouldn't it make more sense to create dedicated `metadata.js` files instead containing _all_ the template fields except for the `template.json` data?

So for example `12-hours-in-barcelona/index.js` would become:

```js
import { default as metadata } from './metadata';
import { default as template } from './template';

export default {
  ...metadata,
  ...template,
};
```

Reply:
yeaup. this makes sense. happy to update